src/server/pps/server/api_server.go

```diff
@@ -2803,6 +2843,7 @@ func (a *apiServer) propagateJobs(txnCtx *txncontext.TransactionContext) error {
 
 		// Check if there is an existing job for the output commit
 		job := client.NewJob(pipelineInfo.Pipeline.Name, txnCtx.CommitSetID)
+		job.Pipeline.Project = pipelineInfo.Pipeline.Project // TODO remove this before merging
```

Comment:
Does this need to be removed?

Reply:
I assumed that you would be introducing this as part of your PR.